pandas/tests/indexes/datetimes/test_constructors.py

```diff
@@ -950,3 +950,60 @@ def test_datetimeindex_constructor_misc(self):
         )
         assert len(idx1) == len(idx2)
         assert idx1.freq == idx2.freq
+
+
+@pytest.mark.parametrize("tz", ["dateutil/Europe/London", "Europe/London"])
+@pytest.mark.parametrize("fold", [0, 1])
+def test_timestamp_constructor_retain_fold(tz, fold):
+    # Test for #25057
+    # Check that we retain fold
+    ts = pd.Timestamp(year=2019, month=10, day=27, hour=1, minute=30, tz=tz, fold=fold)
+    result = ts.fold
+    expected = fold
+    assert result == expected
+
+
+@pytest.mark.parametrize("tz", ["dateutil/Europe/London", "Europe/London"])
+@pytest.mark.parametrize(
+    "value_fold",
```

Comment:
In pytest, you can specify a comma separated string and pass those individual variables in the function so you dont have to index the list

```
@pytest.mark.parametrize('value,fold', [1, 2])
def test_food(self, value, fold):
    ...
```

Reply:
Done.